solrsearch-plugins/src/main/java/co.cask.hydrator.plugin/common/SolrSearchSinkConfig.java

```diff
@@ -116,4 +92,95 @@ public String getIdField() {
     }
     return outputFieldMap;
   }
+
+  /**
+   * Returns the SolrClient for establishing the connection with Solr server, using the hostname and port provided by
+   * user.
+   *
+   * @return Solr client
+   */
+  public SolrClient getSolrConnection() {
+    String urlString;
+    SolrClient solrClient = null;
+    if (solrMode.equals(SINGLE_NODE_MODE)) {
```

Comment:
the UI is not the only way to create a pipeline. Please ignore case when checking.